### PR TITLE
BUGFIX: Fix Breadcrumb Style

### PR DESCRIPTION
**Pull request by crydotsnake**
This PR fixes the bug described in the issue #3474

### CODE REVIEW

File: src/Frontend/Core/Layout/Templates/Breadcrumb.html.twig [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <span class="sr-only">{{ 'lbl.YouAreHere'|trans }}</span>
 <ul class="breadcrumb" itemscope itemtype="https://schema.org/BreadcrumbList" role="navigation">
   {% for breadcrumb in breadcrumb %}
-    <li{% if loop.last %} class="active"{% endif %} itemprop="itemListElement" itemscope
+    <li{% if loop.last %} class="breadcrumb-item active"{% endif %} itemprop="itemListElement" itemscope
```
Comment (carakas) [inline src/Frontend/Core/Layout/Templates/Breadcrumb.html.twig:4]: I think it should be
```suggestion
    <li class="breadcrumb-item{% if loop.last %} active{% endif %}" itemprop="itemListElement" itemscope
```

**Review by carakas — CHANGES_REQUESTED** [review]

File: src/Frontend/Core/Layout/Templates/Breadcrumb.html.twig (hunk 1 above)
Comment (crydotsnake) [reply src/Frontend/Core/Layout/Templates/Breadcrumb.html.twig]: I fixed it!

File: src/Frontend/Core/Layout/Templates/Breadcrumb.html.twig [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <span class="sr-only">{{ 'lbl.YouAreHere'|trans }}</span>
 <ul class="breadcrumb" itemscope itemtype="https://schema.org/BreadcrumbList" role="navigation">
   {% for breadcrumb in breadcrumb %}
-    <li{% if loop.last %} class="active"{% endif %} itemprop="itemListElement" itemscope
+    <li{% if loop.last %} class="breadcrumb-item{% endif %}" itemprop="itemListElement" itemscope
```
Comment (carakas) [inline src/Frontend/Core/Layout/Templates/Breadcrumb.html.twig:4]: it always needs the class breadcrumb-item, and active when it is the last one like I showed in the example

**Review by carakas — CHANGES_REQUESTED** [review]

File: src/Frontend/Core/Layout/Templates/Breadcrumb.html.twig (hunk 2 above)
Comment (crydotsnake) [reply src/Frontend/Core/Layout/Templates/Breadcrumb.html.twig]: Okay. Now I got it!

**Review by carakas — APPROVED** [review]

**Pull request merged**